test/parallel/test-util-inspect.js

```diff
@@ -1367,14 +1367,8 @@ util.inspect(process);
   expect = [
     '{',
     '  a: [Function] {',
-    common.engineSpecificMessage({
-      v8: '    [length]: 0,',
-      chakracore: "    [name]: '',",
-    }),
-    common.engineSpecificMessage({
-      v8: "    [name]: ''",
-      chakracore: '    [length]: 0',
-    }),
+    '    [length]: 0,',
+    "    [name]: ''",
```

Comment:
Is this particular change reverting back to what upstream has? Specifically wondering about the quotation marks

Reply:
It is reverting back, but the quotation marks didn't actually change, right?  They used double quotes for the `name` property because it contains single quotes in it.

Reply:
Yeah I know why it was done that way, I just wasn't sure if upstream node had made the same reasoning or not.

Reply:
Yep, this is what they have upstream.